src/utils/components/Loading/Loading.tsx

```diff
@@ -2,14 +2,18 @@ import React, { FC } from 'react';
 
 import { Bullseye } from '@patternfly/react-core';
 
+import './loading.scss';
+
 const Loading: FC = () => (
-  <Bullseye>
-    <div className="co-m-loader co-an-fade-in-out" data-test="loading-indicator">
-      <div className="co-m-loader-dot__one" />
-      <div className="co-m-loader-dot__two" />
-      <div className="co-m-loader-dot__three" />
-    </div>
-  </Bullseye>
+  <div className="Loading--main">
```

Comment:
Why do we need this classname? This will be good for all the occurrences of this component?

Reply:
for now yes, only setting some height, id amends will be need, they will be amended